.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -25,6 +25,8 @@ the change does.
 <!--- Name of the module/plugin/module/task -->
  - API
  - UI
+ - Installer
```

Comment:
There is no installer is this repo. It may be a good idea to point folks to https://github.com/ansible/awx-operator, or just drop this line in both templates.